### PR TITLE
Make Serving e2e resources uniform

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

**Comment by chaodaiG** [comment]
/lgtm
/approve

This is pretty cool trick to learn

**Comment by mattmoor** [comment]
image

File: config/prod/prow/jobs/config.yaml [hunk 1]
```diff
@@ -217,6 +222,11 @@ presubmits:
           value: /etc/test-account/service-account.json
         - name: E2E_CLUSTER_REGION
           value: us-central1
+        resources:
```
Comment (vaikas) [inline config/prod/prow/jobs/config.yaml:225]: Just curious why not use anchor here as well?

Comment (chizhg) [reply config/prod/prow/jobs/config.yaml:225]: This file is generated, so it does not matter that much to use anchor or not here. More importantly, we are not entirely sure if Prow supports anchors in the Prow job configuration file..

Comment (vaikas) [reply config/prod/prow/jobs/config.yaml:225]: ah, cool cool :) 
Note that we use these kind of anchors in our crd definitions as well, so depending on how they get applied, I'd expect things to just magically to work. But if it's generated, doesn't matter :)  Thanks for the explanation.

**Pull request merged**